### PR TITLE
Add Install documentation and StackStorm workflow install script

### PR DESCRIPTION
**Pull request by joseph-v**
**What this PR does / why we need it**:

This PR adds documentation regarding installation of StackStorm and OpenSDS Workflows. A utility script for automate the installation is also added

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    ```
+* Start docker container using docker-compose
+    ```sh
+    $ docker-compose up -d  # For starting container
+    $ docker-compose down   # For stopping container
+    ```
+* Copy [opensds folder](https://github.com/opensds/orchestration/tree/master/contrib/st2/opensds) to packs folder of stackstorm docker instance.
+	```sh
+	$ docker-compose exec stackstorm /bin/bash
+	# scp -r <host opensds path> /opt/stackstorm/packs/
```
Comment (kumarashit) [inline INSTALL.md]: Will the host opensds path be host:<path of opensds> or is this localhost copying

Comment (kumarashit) [inline INSTALL.md]: Should we use the full path of commands like scp or st2ctl? Like /usr/bin/scp or /usr/bin/st2ctl.

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+OpenSDS Orchestration Dashboard OR CURL may be used for testing
+
+For the examples below OpenSDS is installed on VM with IP: 100.64.41.214 and StackStorm docker image is running on Host IP: 100.64.40.36
+
+* Provision Volume Workflow with input arguments
+ ```sh
+    $ curl -k -X POST \
+        https://100.64.40.36/api/v1/executions \
+        -H  'content-type: application/json' \
+        -H  'X-Auth-Token: c2f427ce9b3d43889ec61eb623160c5d' \
```
Comment (kumarashit) [inline INSTALL.md]: should we give the /auth/v1/tokens POST api too to get the token?

File: scripts/st2_install.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+
+# Input variables
+export ST2_DOCKER_SRC_PATH="/opt/opensds/orchestration"
+export ST2_WORKFLOW_SRC_PATH="/opt/opensds/orchestration/contrib/st2/opensds"
+export HOST_USER=demo_user
+export HOST_PASSWORD=demo_password
+export PACKS_PATH=/opt/stackstorm/packs/
+export HOST_IP=100.64.40.36
```
Comment (kumarashit) [inline scripts/st2_install.sh]: Host IP will be static?

File: INSTALL.md (hunk 1 above)
Comment (joseph-v) [reply INSTALL.md]: Updated this part of code to use named volume

File: scripts/st2_install.sh (hunk 3 above)
Comment (joseph-v) [reply scripts/st2_install.sh]: Updated the script and Host IP is not required

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    ```
+
+* Create Volume using StackStorm Action with input arguments
+    ```sh
+    $ curl -k -X POST   \
+    https://localhost/api/v1/executions   \
+    -H  'content-type: application/json'   \
+    -H  'X-Auth-Token: c2f427ce9b3d43889ec61eb623160c5d'   \
+    -d '{"action": "opensds.create-volume", "user": nul, "parameters": {"ipaddr": "100.64.41.214", "port": "50040", "size": 1, "projectid": "7e515a1edee94f9688efda14b6fb677e", "name": "test101", "token": "gAAAAABcy_WakGraN1iQy8R87ueVwdNYDJP2n9a2J_o7bnptqwFjJLxKMzvJoPVt4ofi74V7kLBCMJzZQ41kcwhWQEOv3v8ne9Z2FrhOOiYPY358Y1-F3gdRTY8oOKlyiqgcAJ9wT5sF5RzqAKwAyinRd3KEkGjsFsjxfFkJSlpRFdJmW_XQ8hwjch539Nwo3RgLaCXX1W0z"}}'
```
[the remark this answers is not in the document]
Comment (joseph-v) [reply INSTALL.md]: Done

**Pull request merged**